app/src/components/datetime/constants.ts

```diff
@@ -7,7 +7,6 @@ export const LAST_N_TIME_RANGES: LastNTimeRange[] = [
   { key: "1d", label: "Last Day" },
   { key: "7d", label: "Last 7 Days" },
   { key: "30d", label: "Last Month" },
-  { key: "all", label: "All Time" },
```

Comment:
Is this due to performance reasons?

Reply:
Yeah I rather just force them to put in a reasonable time range 😓 - can always add it back

Reply:
actually you can set it right now by clearing the time range in custom. Gonna leave for now.